themes/custom/droopler_theme/src/components/d-p-carousel/d-p-carousel.twig

```diff
@@ -21,7 +22,9 @@
           {% block heading %}
           {% endblock %}
 
-          {% include '@droopler_theme/base/divider/divider.twig' %}
+          {% if has_heading %}
```

Comment:
Let's put the rendering of the `heading` block in this condition block.
`
{% if has_heading %}
  {% block heading %}
  {% endblock %}
`

Reply:
Fixed :)